### ISSUE TITLE
Almost always opens GitHub release page upon opening browser

### ISSUE THREAD

**Issue by nistre** [XFox111/TabsAsideExtension#50]
When opening web browser (Edge Canary) the extension often will open the GitHub release page for the current version for TabsAsideExtension.  Verified that my Startup settings don't include the release page as a tab to startup with.

**Comment by XFox111** [comment]
Hi, thanks for your feedback!
I'm sorry about this bug. I wonder why it has appeared. In a mean time, I've temporarily cut this feature out and pushed patched version to the webstores (it will take some time to be processed and approved). However, I probably will remove this feature completely since I have some doubts about its necessity

**Comment by AmineI** [comment]
I think I got why this happens. On browser updates, extensions are sent the "onInstalled" event again. However, Edge Canary being updated almost daily, this is sure to be a bother.

What we can do is tweak the existing code to only open the changelog if the "onInstalled" event was fired because the extension updated.

Let me explain with some [docs](https://developer.chrome.com/extensions/runtime#event-onInstalled) 

The code that was implemented works like this `chrome.runtime.onInstalled.addListener((reason) => showChangelog())`

This `reason` can be one of these : `"install", "update", "chrome_update", or "shared_module_update"`,  according to these [docs](https://developer.chrome.com/extensions/runtime#type-OnInstalledReason)

Since the reason is not checked, the changelog is displayed when the reason is `chrome_update` as well, for example !! So, we just need a switch case on the reason to display the changelog on extension update only :) . 

I'll make a PR !
